### PR TITLE
458 - Create bossy textarea

### LINKED ISSUE #458: Create bossy-textarea

### LINKED ISSUE #459: Unit test for bossy-textarea

### PR DESCRIPTION
**Pull request by tchane24**
Closes #458
Closes #459

### CODE REVIEW

File: test/bossy-textarea.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	let component: BossyInput;
+	beforeEach(async(() => {
+		TestBed.configureTestingModule({declarations: [BossyInput] , })
+			.compileComponents()
+			.then(() => {
+				const fixture = TestBed.createComponent(BossyInput);
+				component = fixture.componentInstance;
+			});
+		}));
+	it('should have a defined component', () => {
```
Comment (tabersays) [inline test/bossy-textarea.spec.ts]: Ideally you check more than if it defined, for a text area, I would also check things like Is there they expected text inside.

**Comment by glawson** [comment]
@tchane24 - fix conflicts please.

File: test/bossy-textarea.spec.ts (hunk 1 above)
Comment (tchane24) [inline test/bossy-textarea.spec.ts]: Alright, I was thinking of having one that makes sure there are rows. I'm not really sure how to check the config variables in the test though.

Comment (glawson) [inline test/bossy-textarea.spec.ts]: @tchane24 - Taber will discuss with you at lab.

**Pull request closed without merging**